spec/requests/red_hat_migration_analytics_manifest_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe "Red Hat  Migration Analytics Manifest API" do
+  let(:api_red_hat_migration_analytics_url) { "#{api_url}/migration_analytics" }
+  let(:manifest) { { "ManageIQ::Providers::Vmware::InfraManager" => {}} }
+
+  describe "POST" do
+    it "/api/red_hat_migration_analytics/:id bundle action" do
+      api_basic_authorize "red_hat_migration_analytics"
+
+      ems1 = FactoryBot.create(:ems_vmware, :name => "sample vmware1")
+
+      allow(Cfme::CloudServices::InventorySync).to receive("bundle_queue").with(@user.userid, manifest, ems1) { 101 }
+      post(api_red_hat_migration_analytics_url(nil, manifest, ems1), :params => { "action" => "bundle" })
```

Comment:
I don't think we want to so a resource url here, otherwise the bundle_collection method won't be called. maybe just post(api_red_hat_migration_analytic_url, ...) with the bundle action and the provider_ids => [ems1.id]